src/routes/login/+page.svelte

```diff
@@ -96,9 +96,10 @@
                 ),
             );
             if (response.ok) {
+                //ToDo fix toast
+                toastStore.trigger(toast);
                 //Login sucessful
```

Comment:
```suggestion
                // ToDo fix toast
                toastStore.trigger(toast);
                // Login sucessful
```
Sollte das `ToDo` da überhaupt noch sein? Sonst bitte entfernen.